### PR TITLE
Integrate 0.10.0-alpha.1 platform SDK

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**

### CODE REVIEW

**Review by tinker-michaelj — APPROVED** [review]
LGTM

File: hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java [hunk 1]
```diff
@@ -125,7 +126,12 @@ public RecordStreamManager(
 					platform,
 					startWriteAtCompleteWindow,
 					RecordStreamType.RECORD);
-			writeQueueThread = new QueueThread<>("writeQueueThread", platform.getSelfId(), streamFileWriter);
+			writeQueueThread = new QueueThreadObjectStreamConfiguration<RecordStreamObject>()
+					.setThreadName("writeQueueThread")
```
Comment (QianSwirlds) [inline hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java:130]: nit: could add a `setComponent("record-stream")`

File: hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java [hunk 2]
```diff
@@ -135,10 +141,12 @@ public RecordStreamManager(
 				new RunningHashCalculatorForStream<>();
 
 		hashCalculator = new HashCalculatorForStream<>(runningHashCalculator);
-		hashQueueThread = new QueueThread<>(
-				"hashQueueThread",
-				platform.getSelfId(),
-				hashCalculator);
+		hashQueueThread = new QueueThreadObjectStreamConfiguration<RecordStreamObject>()
+				.setThreadName("hashQueueThread")
+				.setNodeId(platform.getSelfId().getId())
```
Comment (QianSwirlds) [inline hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java:146]: nit: could add a setComponent("record-stream")

**Comment by tinker-michaelj** [comment]
> LGTM, but the `build-artifact` is failing on CircleCI, so needs to be fixed

Hi Qian, this branch is a proof-of-concept hack of a new NFT service; the unit tests are all horribly broken 😁

**Review by QianSwirlds — APPROVED** [review]

**Comment by QianSwirlds** [comment]
> > LGTM, but the `build-artifact` is failing on CircleCI, so needs to be fixed
> 
> Hi Qian, this branch is a proof-of-concept hack of a new NFT service; the unit tests are all horribly broken 😁

oh, got it. Thank you for the information! :)

**Comment by steven-sheehy** [comment]
Should convert PR to draft to avoid accidental reviews/merges ;)

File: hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java (hunk 2 above)
Comment (Neeharika-Sompalli) [reply hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java:146]: Fixed. Thanks Qian!

File: hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java (hunk 1 above)
Comment (Neeharika-Sompalli) [reply hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java:130]: Fixed. Thanks Qian!

**Comment by Neeharika-Sompalli** [comment]
Closing this PR to be able to profile separately on 0.9.0 and 0.10.0 platform sdk.

**Pull request closed without merging**